CHANGELOG.md

```diff
@@ -5,87 +5,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## v1.0dev - [2022-01-20]
 
-Pipeline has been re-implemented in [Nextflow DSL2](https://www.nextflow.io/docs/latest/dsl2.html)
+First release of [nf-core/metapep](https://nf-co.re/metapep/dev), created based on [nf-core](https://nf-co.re) standards and [nf-core/tools](https://nf-co.re/tools) template version 1.13.1.
 
 ### `Added`
 
```

Comment:
Maybe use the summary list of the `README` or the `output.md` as a start and then just add your points around this? Then it would be easier to understand, what would you think?

Reply:
Good point, I can actually use the readme and add some extra points for development tracking! 👍